### DISCUSSION TITLE
Can't see 'Resources' button / window on my API

### DISCUSSION THREAD

**Discussion by juaninicolai** [microcks/microcks#1243] [Q&A] [answered]
I am following this [tutorial](https://microcks.io/documentation/guides/usage/direct-api/#create-resources) to onboard my API onto Microcks. So far everything has worked. I am able to see my API (Created via Direct Generator) and I have sent some POST request to create extra resources which I am able to retrieve via GET /{id}.

With that said, I can't see on my UI the 'Resources' window / tab (as shown in the tutorial) which is supposed to list all the resources added via POST request.

Has anybody experienced the same or know how I can solve this?

Thanks!

**Comment by lbroudoux** [comment, marked as answer]
Hi @juaninicolai,

It appears that in the latest versions, the screenshot is a bit different with the "Test" box now displayed on top of the "Check what's in the repository" link like shown below:

<img width="569" alt="Capture d’écran 2024-07-16 à 20 42 51" src="https://github.com/user-attachments/assets/373a3f2a-2e18-46d3-a839-00ec5ff1ec80">

Can you see that link, too?

**Reply by juaninicolai** [reply]
Oh cool! Yes, I can see the link and have found the resources. Thank you very much!
